packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -9,6 +9,9 @@ import { EventEmitter } from "events";
 import { IValueFactory, IValueOpEmitter, IValueOperation, IValueType } from "@fluidframework/map";
 import * as MergeTree from "@fluidframework/merge-tree";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
+import { v4 as uuid } from "uuid";
+
+const reservedIntervalIdKey = "intervalId";
```

Comment:
is there a possibily of the user overwirtting this? specifically is there  a way to set properties on an interval after it is created? if so, i think we might want to consider making it immutable, otherwise the merge logic will get complex as clients could have different id's at different reference sequence numbers, which we'd need to track, and correctly resolve if we ever allow operations by id